### DISCUSSION TITLE
Boxed lines instead of curved lines in TREE diagram

### DISCUSSION THREAD

**Discussion by Thoybur-Rohman** [mermaid-js/mermaid#4406] [General]
Issue: i would like the lines to be boxed but Marmaid ( from what iv found ) does not have any option for this ) does anyone know or has done this? 

graph TD
    A[STUFF] -->|Get money| B(Go shopping)
    B ==> C{Let me think}
    C -->|One| D[Laptop]
    C -->|Two| E[iPhone]
    C ==>|Three| F[fa:fa-car Car]

OUTPUT: 
![image](https://github.com/mermaid-js/mermaid/assets/73062879/f5b9312e-5212-4c12-9c7f-c5c3a0ba646a)

**Comment by sidharthv96** [comment]
You can use `flowchart-elk` instead of `graph`.

```mermaid
flowchart-elk TD
A[STUFF] -->|Get money| B(Go shopping)
B ==> C{Let me think}
C -->|One| D[Laptop]
C -->|Two| E[iPhone]
C ==>|Three| F[fa:fa-car Car]
```
